### PR TITLE
feat(health): Report inbound health for Kafka & RabbitMQ

### PR DESCRIPTION
**Pull request by sbuettner**
## Description

Report Kafka inbound connector health.

## Related issues

relates to #621

### CODE REVIEW

**Review by chillleader — APPROVED** [review]
Looks great as a first iteration - although we need to double check how we handle the exceptions in the runtime and make the contract between runtime and connectors clear.

File: connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java [hunk 1]
```diff
@@ -96,27 +83,36 @@ public void setUp() {
...
-    doNothing().when(mockConsumer).assign(any());
-    doAnswer(new AnswersWithDelay(100, new Returns(new ConsumerRecords<>(new HashMap<>()))))
-        .when(mockConsumer)
-        .poll(any());
     KafkaExecutable kafkaExecutable = getConsumerMock();
 
+    when(mockConsumer.partitionsFor(any())).thenReturn(topicPartitions);
+
+    // Return and stop looping
+    when(mockConsumer.poll(any()))
```
Comment (markfarkas-camunda) [inline connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java:91]: Nice adjustment to get rid of the delay, thank you 👍

File: connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java [hunk 2]
```diff
@@ -12,28 +12,17 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.argThat;
-import static org.mockito.Mockito.doAnswer;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.timeout;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
```
Comment (markfarkas-camunda) [inline connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java:15]: Just a small thing: we have an agreement that we don't use wildcard imports. IntelliJ automatically uses wildcards but you can turn this off: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

**Review by markfarkas-camunda — APPROVED** [review]
Looks good to me, just a minor thing with the imports.

File: connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java (hunk 2 above)
Comment (sbuettner) [reply connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java:15]: @markfarkas-camunda Good point. I think it depends on the number of imports but if that has been decided, shouldnt we then reflect this in our spotless config?

Comment (sbuettner) [reply connectors/kafka/src/test/java/io/camunda/connector/kafka/inbound/KafkaExecutableTest.java]: Fixed

**Pull request merged**